app/res/values/strings.xml

```diff
@@ -22,6 +22,7 @@
     <string name="ConnectClaimJobURL">https://%s/api/opportunity/%d/claim</string>
     <string name="ConnectDeliveriesURL">https://%s/api/opportunity/%d/delivery_progress</string>
     <string name="ConnectPaymentConfirmationURL">https://%s/api/payment/%s/confirm</string>
+    <string name="ConnectConfirmUserInvitation">https://connectid.dimagi.com/users/confirm_hq_invite</string>
```

Comment:
Is it possible to define only the relative url to host ? That makes the code more felxible in event we want to have configurable hosts or test environments.